include/net/coap.h

```diff
@@ -360,6 +362,13 @@ u8_t *coap_next_token(void);
 static inline u16_t coap_next_id(void)
 {
 	static u16_t message_id;
+	static int seeded;
```

Comment:
The atomic type should `atomic_t` instead of int, even thou in practice they are the same.

Reply:
done